wiki/Tournaments/MUKASHI/1/en.md

```diff
@@ -0,0 +1,214 @@
+---
+tags:
+  - MUKASHI
+  - MUKASHI SPARK
+---
+
+# osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park
+
+![osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park logo](img/logo.jpg)
+
+The **osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park** was a single-elimination 1v1 osu!mania LAN tournament hosted by [Perkumpulan Orang Santai Total (POST)](https://santai.web.id/) under the guidance and supervision of ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) and ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677) from the Indonesian osu! community. It was the first-ever osu!mania tournament of this scale to be held by POST, who had previously facilitated tournaments for other games such as Tekken 7.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-11-14/2022-11-27 |
+| Technical meeting | 2022-11-30 |
+| Opening | 2022-12-03 (11:00 UTC +7) |
+| Extra Preliminary Match | 2022-12-03 (11:15 UTC +7) |
+| Quarterfinal 1 | 2022-12-03 (10:50 UTC +7) |
+| Quarterfinal 2 | 2022-12-03 (12:25 UTC +7) |
+| Quarterfinal 3 | 2022-12-03 (13:00 UTC +7) |
+| Quarterfinal 4 | 2022-12-03 (13:35 UTC +7) |
+| *Break* | 2022-12-03 (14:00 UTC +7) |
+| Semifinal 1 | 2022-12-03 (14:30 UTC +7) |
+| Semifinal 2 | 2022-12-03 (15:05 UTC +7) |
+| 3rd Place Match | 2022-12-03 (15:40 UTC +7) |
+| Grand Final | 2022-12-03 (16:30 UTC +7) |
+| Closing and awarding | 2022-12-03 (17:00 UTC +7) |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 500,000 IDR, commemorative certificate |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 300,000 IDR, commemorative certificate |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 200,000 IDR, commemorative certificate |
+
+## Organisation
+
+The osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park was run by various POST representatives and Indonesian osu! community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=ID }:: Raben (POST), ::{ flag=ID }:: Ijul (POST), ::{ flag=ID }:: Erlangga (POST) |
```

Comment:
is there any particular reason their profiles aren't linked, although at least two of them [appear](https://osu.ppy.sh/users/31664650) to [exist](https://osu.ppy.sh/users/30984728)?

Reply:
@TicClick those are not "Ijul" and "Erlangga" that are mentioned in the article lol

Reply:
(i genuinely dunno their osu! account or if any of them even have an osu! account tbh)